docs/pages/api-docs/select.md

```diff
@@ -70,6 +70,7 @@ Any other props supplied will be provided to the root element ([Input](/api/inpu
 | <span class="prop-name">iconOpen</span> | <span class="prop-name">.MuiSelect-iconOpen</span> | Styles applied to the icon component if the popup is open.
 | <span class="prop-name">iconFilled</span> | <span class="prop-name">.MuiSelect-iconFilled</span> | Styles applied to the icon component if `variant="filled"`.
 | <span class="prop-name">iconOutlined</span> | <span class="prop-name">.MuiSelect-iconOutlined</span> | Styles applied to the icon component if `variant="outlined"`.
+| <span class="prop-name">shadowInput</span> | <span class="prop-name">.MuiSelect-shadowInput</span> | Styles applied to the shadow input component.
```

Comment:
@oliviertassinari Is it possible to keep classes private? This doesn't seem like a helpful class. At least explain what it is. "shadow input" is unlikely something you understand without knowing the  implementation.

Reply:
We have no infrastructure in place to make a style rule inside a style sheet private. What we could do is add another style hook.

At the very least, I would change it to:

```suggestion
| <span class="prop-name">nativeInput</span> | <span class="prop-name">.MuiSelect-shadowInput</span> | Styles applied to the underlying native input component.
```

*shadow* came from the name we give to the second textarea in `TextareaAutosize`